util/src/main/java/org/hyperledger/besu/util/bytes/BytesValue.java

```diff
@@ -568,6 +568,19 @@ default String getHexString() {
     return toString();
   }
 
+  /** @return This value represented as a minimal hexadecimal string (without any leading zero). */
+  default String toShortHexString() {
```

Comment:
Ok. I have removed the method in `BytesValue`. However i can't wrap directly as a bytes 32 because just before leading zeros are removed so it is not necessarily the right size anymore. Instead i create a `UInt256` from the hexadecimal representation of the value.

Reply:
fixed here: https://github.com/hyperledger/besu/pull/4/commits/ee5a1d1741de00ed32b5c96b3506454208ccf4e1